src/eth/index.ts

```diff
@@ -30,10 +36,24 @@ export class ETH extends EventEmitter {
     this._statusTimeoutId = setTimeout(() => {
       this._peer.disconnect(DISCONNECT_REASONS.TIMEOUT)
     }, ms('5s'))
+
+    // Set forkHash and nextForkBlock
+    if (this._version >= 64) {
+      const c = this._peer._common
+      this._hardfork = c.hardfork() ? (c.hardfork() as string) : this._hardfork
+      this._forkHash = c.forkHash(this._hardfork)
+      // TODO: functionality to get next fork block number could also be integrated into Common
+      const hfBlock: number = c.hardforks().filter((hf: any) => hf.name === 'chainstart')[0].block
+      // Next fork block number or 0 if none available
```

Comment:
@evertonfraga thanks, I think I will open a small PR on this towards Common as well before a release, makes really more sense to add this code along a small `nextForkBlockNumber()` function to Common which we can then reuse.